rebasehelper/helpers/download_helper.py

```diff
@@ -152,6 +152,10 @@ def download_file(url, destination_path, blocksize=8192):
             raise DownloadError(r.reason)
 
         file_size = int(r.headers.get('content-length', -1))
+        # use infinite progress bar in case of compressed content, there is no
+        # way to determine the uncompressed size in advance
```

Comment:
I think the comment should be after the condition, at least it makes more sense to me that way.